arrow-optics/src/test/kotlin/arrow/optics/OptionalTest.kt

```diff
@@ -22,6 +25,25 @@ class OptionalTest : UnitSpec() {
     init {
 
         testLaws(
+
+            OptionalLaws.laws(
```

Comment:
Other instances should also be added since it would've caught the unlawfulness of `listElementOptional` already ;)